### PR TITLE
Update README docs.

### PR DESCRIPTION
**Pull request by mikelehen**
Updates README.md and SETUP.md to remove outdated information and include better instructions for running (parts of) the pipeline locally.  It's not perfect, but hopefully it's a step in the right direction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,45 @@
...
-Recommend virtualenv or miniconda python 3.7 from here
-[https://docs.conda.io/en/latest/miniconda.html](https://docs.conda.io/en/latest/miniconda.html)
-
-If using conda, you can use the following:
-- `conda create python=3.7 -n covid-data-model`
-- `conda activate covid-data-model`
-- `pip install -r requirements.txt -r requirements_test.txt`
-
-### Running Models
+## Running PySEIR
```
Comment (mikelehen) [inline README.md:42]: This PySEIR documentation isn't super useful, but I'm leaving it for now.

File: README.md [hunk 2]
```diff
@@ -1,100 +1,45 @@
...
-`../covid-data-public`.
+It ingests data scraped via https://github.com/covid-projections/can-scrapers, combines it, calculates metrics, and generates data files for the [Covid Act Now API](https://apidocs.covidactnow.org/) and [website](https://covidactnow.org/)
 
+# Development
 
 ## Setup
 
-Detailed setup instructions can be found [here](./SETUP.md)
-
-# API Snapshots
```
Comment (mikelehen) [inline README.md:40]: I deleted the API Snapshots / Labeling snapshots sections since they're kind of out of date and you can kick these off via github UI now. So ideally we'd add instructions for using the github actions UI to kick off snapshots and maybe explain the daily build orchestration, but I didn't feel like doing that in this PR, so I'm just deleting this since it's not super helpful as-is.

File: README.md [hunk 3]
```diff
@@ -1,100 +1,45 @@
...
+# Fetches latest scraped data from can-scrapers and combines all data sources
+# into a combined dataset, runs all filters, etc.  Adding --no-refresh-datasets
+# will make this much faster but skips fetching / combining latest datasets.
+./run.py data update --state CT
 
-# Development
+# Runs the pyseir code to generate the infection rate, i.e. r(t) metric data for locations.
+python ./pyseir/cli.py build-all --states=CT
 
-# Sentry
```
Comment (mikelehen) [inline README.md:66]: I moved this Sentry note to SETUP.md.

File: cli/data.py [hunk 4]
```diff
@@ -120,7 +120,7 @@ def update(
     else:
         multiregion_dataset = timeseries.MultiRegionDataset.from_compressed_pickle(
             dataset_utils.COMBINED_RAW_PICKLE_GZ_PATH
-        )
+        ).get_subset(state=state, fips=fips)
```
Comment (mikelehen) [inline cli/data.py:123]: This was a drive-by improvement to make `./run.py data update` respect the `--state` and `--fips` arguments when you pas `--no-refresh-datasets`.

**Review by smcclure17 — APPROVED** [review]
Looks great, thanks for cleaning this up!
